### PR TITLE
Replace `okta_user_group_memberships` with `okta_group_memberships`

### PR DESCRIPTION
**Pull request by songe**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,12 +9,13 @@ from multiple reviewers, requiring only `one_of` the reviewers to approve the ac
 - Let's say you have an Okta org, and you would like to assign engineers who are on-call to have access to certain apps.
 - You can assign the apps to a user group `Has nice things`, then set up Abbey Quickstart
   - Make sure to [configure your GitHub Actions Secrets](https://docs.abbey.io/product/deploying-your-grant-kit#step-1-add-your-abbey_token) and add `OKTA_API_TOKEN` and `ABBEY_TOKEN`
-- When the user is on call, they can request access to be added to `Has nice things` user group via Abbey. This will create a PR appending the following snippet to `access.tf`:
-  ```hcl
-  resource "okta_user_group_memberships" "has_nice_things__00uSomeOktaUserId" { # replace-me@example.com
-    user_id = "00uSomeOktaUserId"
-    groups = ["00gSomeOktaUserGroupId"]
-  }
+- When the user is on call, they can request access to be added to `Has nice things` user group via Abbey. This will create a PR appending the user ID to the eixsting `okta_group_membership.has_nice_things` resource in `access.tf`:
```
Comment (jeffchao) [inline README.md]: nit: Can we change the group name to something like `On Call` or `Security`?

File: outputs.tf [hunk 2]
```diff
@@ -0,0 +1,4 @@
+resource "okta_group_memberships" "has_nice_things" {
```
Comment (jeffchao) [inline outputs.tf]: What is this one for? It shouldn't be producing any outputs. Did you mean to just put this in `main.tf`?

**Review by jeffchao — APPROVED** [review]
Pre-approving, but requesting some minor changes.

File: outputs.tf (hunk 2 above)
Comment (songe) [reply outputs.tf]: The file was created back when we used `outputs.tf` and `access.tf` interchangeably. Changed the file name to `access.tf`

**Pull request merged**